guides/v2.1/extension-dev-guide/versioning/dependencies.md

```diff
@@ -52,7 +56,6 @@ However, the dependency rules are different:
     },
     ...
 }
```

Comment:
You could indent this block another 4 spaces so that it appears under the bolded text.